src/routes/backoffice.ts

```diff
@@ -502,6 +505,65 @@ export const queueBookingRequestHandler = async (req, res) => {
   }
 };
 
+export const createDirectDebitReturnHandler = async (req, res) => {
+  const { personId, id } = req.params;
+
+  await createDirectDebitReturn(personId, id);
+  res.redirect("back");
+};
+
+export const createDirectDebitReturn = async (personId, id) => {
+  const person = await getPerson(personId);
+  const directDebit = person.transactions.find(
+    (transaction) => transaction.id === id
+  );
+  const directDebitReturnId = directDebit.id.split("-").reverse().join("-");
```

Comment:
❓ What's this conversion for? Is this how the id for a return is normally built? 🤔

Reply:
Normally, id for DD and DDR are random uuid, but in our mock there is convention that DDR has `id` that is reversed `id` of corresponding DD (e.g `dd.id = 'a1-b2-c3'`  and `ddr.id = `c3-b2-a1`).
This convention was already used in other places where we create DDRs, so I've used it here for consistency (although it's not needed anymore because I've added proper field to match DD and DDR: `transaction_id` and `return_transaction_id`)